### PR TITLE
Weekly meeting update; paper statuses updated on main CWP page

### PR DESCRIPTION
**Pull request by graeme-a-stewart**

### CODE REVIEW

File: organization/_posts/2017-10-12-startup.md [hunk 1]
```diff
@@ -0,0 +1,119 @@
...
+- Thursday morning: Plenary summary session for whole workshop
+    - Workshop will close at noon.
+
+A common WLCG/HSF group will finalise the agenda, but everyone can/should suggest topics we should be working on, especially coming out of our CWP working groups. Note - we are easily able to have several HSF focused sessions running in parallel (there are several rooms available).
+
+This approach/suggestion was discussed and well received at GDB yesterday.
+
+### Performance and Costs WG
+
+WLCG Working Group on System Performance and Cost Model has been setup (see [GDB yesterday](https://indico.cern.ch/event/578991/)). Has been going on for a while, convened by Markus Schulz. Idea is to present more in March, so people need to do some work now. It was discussed in the GDB that this group should actually operate at the HSF level.
```
Comment (hegner) [inline organization/_posts/2017-10-12-startup.md]: has been set up (not *setup*)

File: organization/_posts/2017-10-12-startup.md [hunk 2]
```diff
@@ -0,0 +1,119 @@
...
+
+### German Community
+
+Benedikt reported they would like to start a project for upgrading reconstruction software and will remain in contact with the HSF.
+
+## CWP
+
+- #### General matters and Editorial Board
+    - Graeme, Michel, John and Pete are hard at work assembling the synthesis paper
+        - For the moment  working in a shadow document as the process of assembling material and doing at least one consistency check is not useful for others.
```
Comment (hegner) [inline organization/_posts/2017-10-12-startup.md]: remove space between *moment* and *working*

File: organization/_posts/2017-10-12-startup.md [hunk 3]
```diff
@@ -0,0 +1,119 @@
...
+-   #### Workflow and Resource Management
+    - Kaushik is making the first draft now, then passing it around other contributors. Draft available by next week?
+
+-   #### Visualization
+    - No news.
+
+-   #### Event/Data Processing Frameworks
+    - 6 page summary almost ready.
+    - One point about workflow tool and framework interplay to add to the summary document draft.
+    - The full document almost not necessary any more. Will contain more info about the current status.
```
Comment (hegner) [inline organization/_posts/2017-10-12-startup.md]: the conclusion was to forget about the long one for the CWP time scale

File: organization/_posts/2017-10-12-startup.md [hunk 4]
```diff
@@ -0,0 +1,119 @@
...
+
+## General News
+
+### WLCG/HSF Workshop
+
+This has changed from being 2 parallel workshops to a single workshop, with both parallel and plenary sessions for the two communities. [Draft announcement prepared](https://docs.google.com/document/d/18dDKX11gWxDCEb9lagWj9BSlMGAUUHLYUOBuSxYroMo/edit?usp=sharing). Workshop will run from Monday 26 March at 9am until Thursday 29 March at noon.
+
+A straw man agenda:
+- Monday morning: common welcome session with WLCG/HSF
+- Monday morning/afternoon: short presentations by working groups
```
Comment (jouvin) [inline organization/_posts/2017-10-12-startup.md]: You are lacking a blank line here, thus the bullet list is interpreted as part of the § "a straw man....". Not sure why pandoc didn't make it right...

File: organization/_posts/2017-10-12-startup.md [hunk 5]
```diff
@@ -0,0 +1,119 @@
...
+            - Scope and Challenges
+            - Current Practice
+            - Research and Development Program
+              - ~3 Year: What would we like to understand to make decisions for the ATLAS and CMS Computing TDRs in ~2020? Which should explain how we might actually do and afford HL-LHC computing.
+              - ~5 year: What elements needed for HL-LHC could we deploy already in LHC Run 3 to gain experience and what might these elements look like?
+            - Discussion: timescales are not completely rigid, more for the road map milestones, though these are useful for the LHCC. 1 year goals are *still things we need to do* - don’t remove them from the individual papers! They will be merged in the 3 year timescales as needed for the synthesis paper (but that paper should not have sections that go out of date as quickly as a year).
+        - So it’s very useful if document(s) address those points directly, otherwise this has to be "translated".
+    - We should identify papers needing more feedback and publicise this to the community.
+    - For papers in good shape, they should go to archive by 31 October (indeed, put them in now if possible).
```
Comment (hegner) [inline organization/_posts/2017-10-12-startup.md]: arxiv, not archive

File: organization/_posts/2017-10-12-startup.md [hunk 6]
```diff
@@ -0,0 +1,119 @@
...
+- Monday morning: common welcome session with WLCG/HSF
+- Monday morning/afternoon: short presentations by working groups
+- Tuesday morning: common session with WLCG/HSF
+- Tuesday afternoon: available for work group sessions
+- Wednesday morning: common session with WLCG/HSF
+- Wednesday afternoon: available for work group sessions
+- Thursday morning: Plenary summary session for whole workshop
+    - Workshop will close at noon.
+
+A common WLCG/HSF group will finalise the agenda, but everyone can/should suggest topics we should be working on, especially coming out of our CWP working groups. Note - we are easily able to have several HSF focused sessions running in parallel (there are several rooms available).
```
Comment (hegner) [inline organization/_posts/2017-10-12-startup.md]: we discussed the deadline for when to deliver a first agenda. should be added

**Comment by graeme-a-stewart** [comment]
Thanks @hegner - I would suggest using "squash and merge" on these PRs rather than "rebase and merge". The fixup commits are rather uninteresting.

**Pull request merged**